shell/platform/darwin/ios/framework/Source/FlutterPlatformViews.mm

```diff
@@ -1035,6 +1041,14 @@ - (void)touchesCancelled:(NSSet<UITouch*>*)touches withEvent:(UIEvent*)event {
 - (void)touchesEnded:(NSSet*)touches withEvent:(UIEvent*)event {
 }
 
+- (void)setFlutterAccessibilityContainer:(NSObject*)flutterAccessibilityContainer {
```

Comment:
Why is this being declared as an ivar and manual getter/setter methods, rather than declared as a property, with synthesized methods?

Reply:
We don't need the getter, we only need the setter so the "hidden" getter, accessibilityContainer used by UIAccessibility can return the correct value.